tests/integration/kubernetes/gha-run.sh

```diff
@@ -241,13 +242,90 @@ function cleanup() {
 
 function deploy_snapshotter() {
 	echo "::group::Deploying ${SNAPSHOTTER:-}"
-	#TODO Add the deployment logic for the snapshotter in PR https://github.com/kata-containers/kata-containers/pull/8585.
+	case ${SNAPSHOTTER:-} in
+		nydus) deploy_nydus_snapshotter ;;
+		*) >&2 echo "${SNAPSHOTTER:-} flavour is not supported"; exit 2 ;;
+	esac	
 	echo "::endgroup::"
 }
 
 function cleanup_snapshotter() {
 	echo "::group::Cleanuping ${SNAPSHOTTER:-}"
-	#TODO Add the logic for cleaning up the snapshotter in PR https://github.com/kata-containers/kata-containers/pull/8585.
+	case ${SNAPSHOTTER:-} in
```

Comment:
I'd prefer having SNAPSHOTTER declared atop, instead of having SNAPSHOTTER:- everywhere.